server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/api/posttask/PostProjectAnalysisTasksExecutor.java

```diff
@@ -183,8 +209,17 @@ public QualityGate getQualityGate() {
     }
 
     @Override
+    @Deprecated
     public Date getDate() {
```

Comment:
this is done in the interface
I had added the annotation on the implementation, but it's actually useless and I've removed it